revenuecat_examples/purchase_tester/lib/src/upsell.dart

```diff
@@ -285,9 +285,10 @@ class _PurchaseButton extends StatelessWidget {
 
   Future<void> _purchasePackage(BuildContext context, Package package) async {
     try {
-      final customerInfo = await Purchases.purchasePackage(package);
-      final isPro =
-          customerInfo.entitlements.active.containsKey(entitlementKey);
+      final purchaseResult = await Purchases.purchasePackage(package);
+      final isPro = purchaseResult.customerInfo.entitlements
+        .active.containsKey(entitlementKey);
+      print("StoreTransaction: ${purchaseResult.storeTransaction}");
```

Comment:
Just to confirm. Is this print intended to be commited?

Reply:
I would like to leave it as I think it can be useful... This is the test app in any case so I think it's ok